sys/mips/conf/std.CHERI_PURECAP

```diff
@@ -0,0 +1,34 @@
+#
+# std.CHERI_PURECAP -- Purecap CHERI Kernel variant base configuration
+#
+# $FreeBSD$
+#
+
+include		"std.CHERI"
+
+machine		mips mips64c128
+
+nomakeoptions	ARCH_FLAGS
+makeoptions 	ARCH_PIC_FLAGS="-fpic"
+makeoptions 	ARCH_FLAGS="-mcpu=cheri128 -mxcaptable"
+makeoptions 	CFLAGS+="-gdwarf-5"
+makeoptions 	WERROR="-Wno-error"
+makeoptions 	KERNLOADADDR=0xffffffff80100000
+makeoptions 	CHERI_USE_CAP_TABLE=plt
+makeoptions	CHERI_SUBOBJECT_BOUNDS=subobject-safe
+makeoptions	LDSCRIPT_NAME=ldscript.mips.cheri
+
+# purecap kernel options
+options		CHERI_PURECAP_KERNEL
+
+# debugging
+options		KDB_TRACE # stacktrace on panic
+nooptions	DDB
```

Comment:
Is this disabled because it doesn't work? We'll want to fix that soon.

Reply:
It works on RISC-V.  I have not looked at what it would take to fix it on MIPS, probably not too much further work.

Reply:
It has been untested for a long time, probably needs some work.